lib/Humming-Bird/Core.rakumod

```diff
@@ -259,20 +259,20 @@ class Response is HTTPAction is export {
     }
 
     # Write a string to the body of the response, optionally provide a content type
-    method write(Str $body, Str $content_type = 'text/plain', --> Response) {
+    method write(Str:D $body, Str:D $content_type = 'text/plain', --> Response) {
```

Comment:
This isn't a modification but that comma in front of the return arrow looks strange to me. I wonder what purpose it serves or how it works.

Reply:
It doesn't actually do anything. I just like it as a separator on optional args and return type.